src/views/documents/utils/QueryItems.vue

```diff
@@ -17,17 +17,12 @@
         <span slot="button" class="button is-size-7-mobile" :disabled="category.fields.length === 0">
           <fa-icon :icon="$options.categoryIcon[category.name]" />
           <span class="is-hidden-mobile">
-            <!-- $gettext('General') -->
-            <!-- $gettext('ratings') -->
-            <!-- $gettext('Terrain') -->
-            <!-- $gettext('Miscs') -->
```

Comment:
Is thee a specific reason for removing these comments? They are i18n-related, and should not be removed (except if unused). See https://github.com/c2corg/c2c_ui/wiki/i18n

Reply:
Ok my bad, I thought it was dead code